py/occam_base/model.py

```diff
@@ -1,12 +1,18 @@
-import occam
+from enum import Enum
+
+
+class ModelType(Enum):
+    TOP = 'top'
+    BOTTOM = 'bottom'
+    DEFAULT = 'default'
 
 
 class Model:
     """
     Wrapper class for Model
     """
 
-    def __init__(self, ref=None):
+    def __init__(self, ref=None) -> None:
```

Comment:
Does python2 support typing?

Reply:
It doesn't. But everything from occam_base will only be used with python 3 versions of fit, sbsearch, etc